### DISCUSSION TITLE
For updated Latest version

### DISCUSSION THREAD

**Discussion by Tirth1110** [OrchardCMS/OrchardCore#14562] [General]
Hello, 
Today I have take latest repo from **https://github.com/OrchardCMS/OrchardCore.git** this link and you all know .Net 8.0 is not released officially from Microsoft. 

Just today I have started Learning Orchard Framework and I am getting issues when I build my Orchard application.
it's showing error like below.

Severity	Code	Description	Project	File	Line	Suppression State
Error	NETSDK1209	The current Visual Studio version does not support targeting .NET 8.0.  Either target .NET 7.0 or lower, or use Visual Studio version 17.8 or higher	OrchardCore.ResourceManagement.Core	C:\Program Files\dotnet\sdk\7.0.402\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets	165	

![image](https://github.com/OrchardCMS/OrchardCore/assets/59839553/4678645f-5c2e-4e76-9569-3fb2aef7b23a)

Please help us to resolve this problem

**Comment by Tirth1110** [comment]
![image](https://github.com/OrchardCMS/OrchardCore/assets/59839553/fabd773e-d93c-4234-8035-2ee9f32d33dc)

In my system, I have installed Net 6.0 and .Net 7.0 is available. .Net 8.0 is preview it's not officially released by Microsoft

**Comment by jtkech** [comment]
You could use a preview version of Visual Studio but as I remember in a regular version you have an option to use preview frameworks, hmm but dotnet 8.0 is publicly in the rc2 version so maybe just try to download it.

And then install the dotnet 8.0 sdk and runtime.

Or in the `Dependencies.AspNetCore.props` file set the `DefaultTargetFramework` to `net7.0`

**Comment by jtkech** [comment]
https://dotnet.microsoft.com/en-us/download/dotnet/8.0
